src/web/handlers/PanGestureHandler.ts

```diff
@@ -444,7 +447,7 @@ export default class PanGestureHandler extends GestureHandler {
       if (this.activateAfterLongPress > 0) {
         this.activationTimeout = setTimeout(() => {
           this.activate();
-        }, this.activateAfterLongPress);
+        }, this.activateAfterLongPress) as unknown as number;
```

Comment:
Same

Reply:
Done in [f6a46e5](https://github.com/software-mansion/react-native-gesture-handler/pull/2873/commits/f6a46e59bf7ebe489416d68fc2700e18d5bb3d05)